test/e2e/specs/dashboard/settings__site-visibility.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { test, expect } from '../../lib/pw-base';
+
+test.describe( 'Dashboard: Site Visibility Settings', { tag: '@dashboard' }, () => {
+	test( 'As a new simple site user, I can set my site visibility to Private, so that only I can see my site', async ( {
+		helperData,
+		page,
+		pageDashboard,
+		pageIncognito,
+		sitePublic,
+	} ) => {
+		const incognitoContentLocator = pageIncognito.locator( 'html' );
+
+		await test.step( "Given I am on my new public site's visibility settings page", async function () {
+			// TODO: This should use a DashboardPage method to navigate to the correct page
+			// but it doesn't work for WordPress.com as it expects /v2 to be available.
+			await page.goto(
+				helperData.getCalypsoURL(
+					`sites/${ sitePublic.blog_details.site_slug }/settings/site-visibility`
+				)
+			);
+		} );
+
+		await test.step( "When I set my site's visibility to Private", async function () {
+			await pageDashboard.setSiteVisibility( 'Private' );
+		} );
+
+		await test.step( "And I save my site's visbility settings", async function () {
+			expect( await pageDashboard.saveSiteVisibilityChanges() ).toBe(
+				'Site visibility settings saved.'
```

Comment:
@p-jackson  what do you think of asserting the returned snackbar text in the spec?